CONTRIBUTING.md

```diff
@@ -0,0 +1,6 @@
+Guidelines for contributing can be found locally in this repository at
```

Comment:
Does this need to change to markdown? (seems unrelated to the PR current description)

Reply:
I thought markdown would be more readable, but actually RST does look nicer.